ci/scripts/tests/workflow.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0"?>
```

Comment:
Is there a reason to commit this file? Especially since the comment says, the workflow was automatically generated at ....